tools/GSDumpGUI/GSDumpGUI.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>GSDumpGUI</RootNamespace>
     <AssemblyName>GSDumpGUI</AssemblyName>
-    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
```

Comment:
Just some info:
From what I have understood, 4.6.0 is the latest (or last) version supported by Vista & Windows Server 2008.
https://docs.microsoft.com/en-us/dotnet/framework/install/on-windows-vista

Reply:
Well if by some miracle someone still uses Vista they don't have to worry since the requirement is 4.0 so it's all good.

Reply:
Actually I consider to port it to wpf/netcore3.0 once it is available. Then it would as well run natively under linux. I wonder how gregory is using it tbh.